lib/models/project.js

```diff
@@ -616,20 +616,45 @@ Project.closest = function(pathName, _ui, _cli) {
  */
 Project.closestSync = function(pathName, _ui, _cli) {
   var ui = ensureUI(_ui);
-
-  try {
-    var directory = findup.sync(pathName, 'package.json');
-    var pkg = require(path.join(directory, 'package.json'));
-
-    if (pkg && pkg.name === 'ember-cli') {
-      return Project.nullProject(_ui, _cli);
+  var directory, pkg;
+  
+  // special lookup rules only when testing
+  if (_cli && _cli.testing) {
+      directory = existsSync(path.join(pathName, 'package.json')) && process.cwd();
```

Comment:
so the idea is, if we are testing, and their is a pathName/package.json, we should instead prefer process.cwd(); ?